i18n/nl.toml

```diff
@@ -0,0 +1,24 @@
+[category]
+other = "Categorie"
+
+[tag]
+other = "Label"
+
+[series]
+other = "Serie"
+
+[reading_time]
+one = "Een minuut leestijd"
+other = "{{ .Count }} minuten leestijd"
```

Comment:
Have you tested this translation in posts with a reading time longer than one minute?
Some languages, like Russian, need extra configuration for this translation.

Reply:
Yes, I have tested with both a short and longer post and both look good to me.
(BTW, the "one" entry is missing in the English translation?)

Reply:
Yup. It is missing or it was removed unintentionally.